backend/controllers/feedController.js

```diff
@@ -193,6 +194,24 @@ const getUserInfo = async (req, res, next) => {
 };
 
 const getAllUsersInfo = async (req, res, next) => {
+  const { search, orderBy } = req.query;
+  let searchQuery;
+  let orderByQuery;
+  if (search.length > 0) {
+    searchQuery = {
+      [Op.or]: {
+        first_name: { [Op.startsWith]: search },
+        last_name: { [Op.startsWith]: search },
+      },
+    };
+  } else {
+    searchQuery = { first_name: { [Op.ne]: null } };
+  }
```

Comment:
- use `like` clause for wildcard searching purposes. If you use `startsWith` you only search a text that starts with. By `like` you can search in any position e.g. starts, end, or even in between.
- and by using `like` you can avoid also adding conditioning, by using `??` operator also